src/main/java/com/scottlogic/Main.java

```diff
@@ -8,29 +11,32 @@
 
     public static void main(String[] args) {
         List<UserPost> userPosts = getUserPosts();
-        printAllUserPosts(userPosts);
-        printUserPostsByAscendingAuthor(userPosts);
+        sortAuthorAscending(userPosts);
+        sortAuthorDescending(userPosts);
     }
 
     /**
-     * Prints out all user posts
+     * Prints out user posts in Author-ascending order
      * @param userPosts The list of user posts to print out
      */
-    private static void printAllUserPosts(List<UserPost> userPosts) {
+    private static void sortAuthorAscending(List<UserPost> userPosts) {
+        SortStrategy sortAuthorAscending = new SortStrategy(new SortAuthorAscending());
```

Comment:
I find it odd you are instantiating your sort strategy in main and not your sorting class

Reply:
How would I go about instantiating it into the sorting class?